src/css/cluster-list.css

```diff
@@ -0,0 +1,94 @@
+.add-node-wrapper {
+    position: sticky;
+    top: 0;
+    z-index: 1000;
+    background-color: white;
+    width: 100%;
+    margin-top: 0;
+    text-align: right;
+    padding: 16px 12px 0 12px;
+}
+
+.cluster-list-modal {
+    padding: 0;
+}
+
+.modal-header {
+    padding: 15px 15px 0 15px;
+}
+
+.table-responsive {
+    padding: 0 12px 12px 12px;
+}
+
+.labels-row {
+    th {
+        border-top: none;
+    }
+}
+
+@media (min-width: 1440px) {
+    .modal-dialog.modal-lg {
+        width: 1400px;
+        max-width: 1400px;
+    }
+}
+
+.errors {
+    min-height: 36px;
+}
+
+.table td, .table th {
```

Comment:
Please, try to always scope selectors under the directive's root element selector. Directives in angularjs does not use shadow dom, neither a virtual one, so in theory, its possible to get some styles overlapping